src/components/MapInterface/MapInterface.tsx

```diff
@@ -9,13 +9,17 @@ import { BackgroundLegend, SelectBackground } from './background'
 import { SelectTheme } from './SelectTheme'
 import { SelectTopics } from './SelectTopics'
 import { SelectDatasets } from './SelectDatasets/SelectDatasets'
+import { ShowOsmNotes } from './ShowOsmNotes'
 
 export const MapInterface: React.FC = () => {
   return (
     <MapProvider>
       <div className="relative flex h-full w-full flex-row gap-4">
         <Map />
-        <SelectTheme />
+        <div className="absolute top-3 left-5 z-10 flex">
```

Comment:
```suggestion
        <div className="absolute top-3 left-5 z-10 flex gap-2">
```

lieber hier als unten als mr-2, da hier generisch

Reply:
Danke, dachte `gap` gibts nur bei grids